Prow action fails to remove lifecycle label

For example, following a comment on #550 we get [this run](https://github.com/llm-d/llm-d-inference-scheduler/actions/runs/22616054969/job/65529103371) error:
```console
Run echo "Removing 'stale' label from issue #550"
  echo "Removing 'stale' label from issue #550"
  gh issue edit 550 --remove-label "lifecycle/stale,lifecycle/rotten"
  shell: /usr/bin/bash -e {0}
  env:
    GITHUB_TOKEN: ***
Removing 'stale' label from issue #550
failed to update https://github.com/llm-d/llm-d-inference-scheduler/pull/550: GraphQL: Resource not accessible by integration (removeLabelsFromLabelable)
failed to update 1 issue
Error: Process completed with exit code 1.
```

--- a/.github/workflows/unstale.yaml
+++ b/.github/workflows/unstale.yaml
@@ -1,20 +1,32 @@
-name: 'Unstale Issue'
+name: 'Unstale Issue or PR'
 
 on:
   issues:
     types: [ reopened ]
   issue_comment:
     types: [ created ]
+  pull_request:
+    types: [ reopened ]
 
 jobs:
   remove-stale:
     runs-on: ubuntu-latest
     permissions:
       issues: write
+      pull-requests: write
     if: >-
-      github.event.issue.state == 'open' &&
-      (contains(github.event.issue.labels.*.name, 'lifecycle/stale') || 
-       contains(github.event.issue.labels.*.name, 'lifecycle/rotten'))
+      (
+        github.event_name == 'pull_request' &&
+        github.event.pull_request.state == 'open' &&
+        (contains(github.event.pull_request.labels.*.name, 'lifecycle/stale') ||
+         contains(github.event.pull_request.labels.*.name, 'lifecycle/rotten'))
+      ) ||
+      (
+        github.event_name != 'pull_request' &&
+        github.event.issue.state == 'open' &&
+        (contains(github.event.issue.labels.*.name, 'lifecycle/stale') ||
+         contains(github.event.issue.labels.*.name, 'lifecycle/rotten'))
+      )
     steps:
       - name: 'Checkout repository'
         uses: actions/checkout@v6
@@ -22,6 +34,8 @@ jobs:
       - name: 'Remove stale labels'
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          TARGET_NUMBER: ${{ github.event.issue.number || github.event.pull_request.number }}
         run: |
-          echo "Removing 'stale' label from issue #${{ github.event.issue.number }}"
-          gh issue edit ${{ github.event.issue.number }} --remove-label "lifecycle/stale,lifecycle/rotten"
+          echo "Removing stale labels from item #$TARGET_NUMBER"
+          gh issue edit "$TARGET_NUMBER" --remove-label "lifecycle/stale" || true
+          gh issue edit "$TARGET_NUMBER" --remove-label "lifecycle/rotten" || true

